endtoend/beacon_node.go

```diff
@@ -46,6 +48,8 @@ func startBeaconNodes(t *testing.T, config *end2EndConfig) []*beaconNodeInfo {
 		newNode := startNewBeaconNode(t, config, nodeInfo)
 		nodeInfo = append(nodeInfo, newNode)
 	}
+	// Small buffer to make logging cleaner at a glance.
+	t.Log("")
```

Comment:
you dont really need to do this, on the first log you can add in a new line at the end as an escape char `\n`